#! /bin/sh /usr/share/dpatch/dpatch-run
## 25_standard_medium.dpatch by Roland Rosenfeld <roland@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Change the global default from standard.Cautious to
## DP: standard.Medium, which is similar to the old 3.0.3 behavior and 
## DP: doesn't change too much.

@DPATCH@
diff -urNad privoxy~/global.action privoxy/global.action
--- privoxy~/global.action
+++ privoxy/global.action
@@ -10,29 +10,29 @@
 -crunch-outgoing-cookies \
 -crunch-incoming-cookies \
 -crunch-server-header \
--deanimate-gifs \
++deanimate-gifs{last} \
 -downgrade-http-version \
 -fast-redirects \
 -filter{js-annoyances} \
 -filter{js-events} \
 -filter{html-annoyances} \
 -filter{content-cookies} \
--filter{refresh-tags} \
++filter{refresh-tags} \
 -filter{unsolicited-popups} \
 -filter{all-popups} \
--filter{img-reorder} \
--filter{banners-by-size} \
++filter{img-reorder} \
++filter{banners-by-size} \
 -filter{banners-by-link} \
--filter{webbugs} \
++filter{webbugs} \
 -filter{tiny-textforms} \
--filter{jumping-windows} \
++filter{jumping-windows} \
 -filter{frameset-borders} \
 -filter{demoronizer} \
 -filter{shockwave-flash} \
 -filter{quicktime-kioskmode} \
 -filter{fun} \
 -filter{crude-parental} \
--filter{ie-exploits} \
++filter{ie-exploits} \
 -filter{site-specifics} \
 -filter{google} \
 -filter{yahoo} \
@@ -52,17 +52,17 @@
 -hide-if-modified-since \
 +hide-forwarded-for-headers \
 +hide-from-header{block} \
--hide-referrer \
++hide-referrer{forge} \
 -hide-user-agent \
 -inspect-jpegs \
 -kill-popups \
 -limit-connect \
--prevent-compression \
++prevent-compression \
 -overwrite-last-modified \
 -redirect \
 -send-vanilla-wafer \
 -send-wafer \
--session-cookies-only \
++session-cookies-only \
 +set-image-blocker{pattern} \
 -treat-forbidden-connects-like-blocks \
 }
